src/components/Prompt.tsx

```diff
@@ -118,31 +142,57 @@ export function Prompt({
                 </Center>
               ) : (
                 <>
+                  <Button
+                    display={hoveredNodeId === node.id ? "block" : "none"}
+                    onClick={() => setIsEditing(!isEditing)}
+                    position="absolute"
+                    top={1}
+                    right={1}
+                    zIndex={10}
+                    variant="outline"
+                    border="0px"
+                    p="1"
+                  >
+                    {(isEditing && hoveredNodeId === node.id) ? <CheckIcon boxSize={4} /> : <EditIcon boxSize={4} />}
+                  </Button>
```

Comment:
<img width="542" alt="Screenshot 2023-03-30 at 3 52 52 PM" src="https://user-images.githubusercontent.com/26209401/228982258-bbd81f5c-1b5c-4dfb-99e2-d32a254d04aa.png">

hmmmm even tho this is position=absolute it seems to be on the same plane as the textbox?

Reply:
also lets get rid of the grey background on hover

Reply:
> hmmmm even tho this is position=absolute it seems to be on the same plane as the textbox?

Not 100% sure what you mean, but I added a margin to the textbox so the edit button doesn't overlap with any underlying text. If we are ok with it overlapping I can remove the margin

Reply:
ah ok yeah hmmm

like ideally we could just add a margin at the lil corner it occupies and not the whole right side but seems difficult

maybe lets just shrink that margin a bit? feels a little excessive atm

Reply:
e.g. lets get rid of this red space

![Uploading image.png…]()

Reply:
e.g. lets get rid of this red space

![image](https://user-images.githubusercontent.com/26209401/228985960-e481f1f5-7514-4c0e-b479-832371cc921c.png)

Reply:
Done!